mods/noita-mp/files/scripts/noita-components/name-tags.lua

```diff
@@ -0,0 +1,23 @@
+
+function PlayerNameFunction(playerName)
+    gui = gui or GuiCreate()
+    GuiStartFrame(gui)
+    local x1, y1 = GuiGetScreenDimensions(gui)
```

Comment:
Just naming stuff and I dont want to tease you:

Might be better to have more readable names, instead of x1 and y1.

You can decide, if you want to change this :)